packages/components/src/theme.ts

```diff
@@ -4,7 +4,7 @@ export const colors = {
   orange: "#FF8A00",
   fail: "#DD1539",
   white: "#fff",
-  black: "#0B0B0B",
+  black: "#0F0F0F",
   gray: "#292929",
   rblack: "#000000",
```

Comment:
should we delete this?

Reply:
Ops, definitely

Reply:
Hmm I would say to not delete it just in case we need to switch to a black black somewhere (like some text or something)